terragrunt/run-all/large-plan/a/main.tf

```diff
@@ -0,0 +1,3 @@
+resource "null_resource" "null" {
+  count = 5000
```

Comment:
would be nice to have this as a variable, with 5k as a default, just so we can play with it.